packages/fether-react/src/Send/SignedTxSummary/SignedTxSummary.js

```diff
@@ -79,20 +79,21 @@ class SignedTxSummary extends Component {
                       <form className='send-form' onSubmit={handleSubmit}>
                         <fieldset className='form_fields'>
                           <Field
-                            className='form_field_amount'
+                            as='textarea'
+                            className='form_field_value'
+                            value={tx.to}
```

Comment:
The values are already set in initialValues a couple of lines above, so we don't need value here